vrecord

```diff
@@ -1054,6 +1074,7 @@ VIDEO_INPUT_OPTIONS=("Composite" "SDI" "Component" "S-Video")
 AUDIO_INPUT_OPTIONS=("Analog" "SDI Embedded Audio" "Digital Audio (AES/EBU)")
 CONTAINER_OPTIONS=("QuickTime" "Matroska" "AVI" "MXF")
 VIDEO_CODEC_OPTIONS=("Uncompressed Video" "FFV1 version 3" "JPEG2000" "ProRes" )
+AUDIO_CODEC_OPTIONS=("24-bit PCM" "FLAC")
```

Comment:
Might want to note the bit depth of the FLAC setting as well to avoid confusion/ambiguity? If I am reading the code correctly that should also be defaulting to 24 from the decklink capture settings, right?